src/test/java/seedu/address/logic/commands/contact/CEditCommandTest.java

```diff
@@ -37,10 +39,11 @@
     @Test
     public void execute_allFieldsSpecifiedUnfilteredList_success() {
         Contact editedContact = new PersonBuilder().build();
-        EditContactDescriptor descriptor = new EditContactDescriptorBuilder(editedContact).build();
+        EditContactDescriptor descriptor = new EditContactDescriptorBuilder(editedContact,
+            null, true).build();
         CEditCommand cEditCommand = new CEditCommand(INDEX_FIRST_PERSON, descriptor);
 
-        String expectedMessage = String.format(cEditCommand.MESSAGE_EDIT_CONTACT_SUCCESS, editedContact);
+        String expectedMessage = String.format(CEditCommand.MESSAGE_EDIT_CONTACT_SUCCESS, editedContact);
```

Comment:
should we keep it to cEditCommand.MESSAGE_EDIT_CONTACT_SUCCESS? I think AB3 implemented that way previously